src/planeta/cooldown.js

```diff
@@ -0,0 +1,26 @@
+import { getStoredValue, storeValues } from "../services/localStorage";
+import getConfig from "../config";
+
+// COOLDOWN is in seconds, here we convert it to millisecods
+// so we don't need to do extra conversions.
+const COOLDOWN = getConfig().PLANET_A.COOLDOWN * 1000;
+
+const getCooldownFor = passport => JSON.parse(getStoredValue("cooldown", passport) || "{}");
+
+export function addHandshake(myPassport, theirPassport) {
```

Comment:
The method is called addHandshake, but we don't really adding anything. Also `theirPassport` argument is not used here. Was it "by design"?

Reply:
`theirPassport` is used :)